docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.7'
+
+services:
+  docs:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - 8000:8000
+    volumes:
+      - .:/workspaces/a-guide-to-mlops
+    command: poetry run mkdocs serve --dev-addr 0.0.0.0:8000
```

Comment:
Out of curiosity, is it necessary to specify the `poetry run` part? Is this because Poetry is installed separated from system interpreter and `mkdocs` command cannot thus be used directly?

Reply:
Yes this is necessary